uvm/servlets/admin/config/administration/Main.js

```diff
@@ -22,7 +22,7 @@ Ext.define('Ung.config.administration.Main', {
         { xtype: 'config-administration-admin' },
         { xtype: 'config-administration-certificates' },
         { xtype: 'config-administration-snmp' },
-        { xtype: 'config-administration-skins' },
+        //{ xtype: 'config-administration-skins' },
```

Comment:
Let's just delete it, not comment it out.

Reply:
removed